test/test_compiler.py

```diff
@@ -46,11 +45,49 @@ def test_multi_table_delete_multiple(self):
         )
 
     def test_multi_table_update(self):
-        statement = table1.update()\
-                    .values(name=table2.c.name)\
-                    .where(table1.c.id == table2.c.name)
+        statement = table1.update() \
+            .values(name=table2.c.name) \
+            .where(table1.c.id == table2.c.name)
         self.assert_compile(
             statement,
             "UPDATE table1 SET name=table2.name FROM table2 "
             "WHERE table1.id = table2.name"
         )
+
+
+def test_quoted_name_label(engine_testaccount):
+    # quote label
+    col = column('colname').label(quoted_name('alias', True))
+    sel_from_tbl = select([col]).group_by(col).select_from(
+        table(quoted_name('abc', True)))
+    compiled_result = sel_from_tbl.compile(engine_testaccount)
+
+    assert str(compiled_result) == """SELECT colname AS "alias" 
+FROM "abc" GROUP BY colname"""
+
```

Comment:
For the sake of clarity, I would not quote the table name in this case:

```python
    # quote label
    col = column('colname').label(quoted_name('alias', True))
    sel_from_tbl = select([col]).group_by(col).select_from(
        table('abc'))
    compiled_result = sel_from_tbl.compile(engine_testaccount)
     assert str(compiled_result) == """SELECT colname AS "alias" 
FROM abc GROUP BY colname"""
```

Reply:
ok, but your test case in #57 did the quote the table name?

Reply:
Yeah, I don't remember exactly what my reasoning was, but the test case was written in just a few minutes and I think my point was to show that it was quoting the table name, but not the label. But to keep things as modular as possible, I would keep the label quoting test cases separate from any potential table name quoting tests.

Reply:
Fair enough.